File: w01/d04/John_Murphy/calculator/calculator.rb

class Calculator

  def add(x, y)
    x + y
  end

  def subtract(x, y)
    x - y
  end

  def divide(x, y)
    return "NaN" if y == 0
    x.to_f/y
  end

  def multiply(x, y)
    x*y
  end

end

File: w06/d01/Nessa_Nguyen/calculator.js
function Calculator(num1, num2, operation) {
  var result = 0;
  if (operation == 'add') {
    result = num1 + num2;
  } else if (operation == 'subtract') {
    result = num1 - num2;
  } else if (operation == 'multiply') {
    result = num1 * num2;
  } else if (operation == 'divide') {
    result  = num1 / num2;
  } else {
    result = "That's not a math operation";
  }
  
  console.log(result);  
}

Calculator(10, 77, 'multiply');
Calculator(10, 77, 857);
